Assets/HotmaxEditorVR/Hittables/deletePanel.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Events;
+
+
+public class deletePanel : MonoBehaviour, IHittable
+{
+    //--local refs
+    private stateManager _stateManagerMutatorRef;
+    private GameObject _selectedObject;
+
+    public Transform target;
+
+    private void Awake()
+    {
+        _stateManagerMutatorRef = GameObject.FindObjectOfType(typeof(stateManager)) as stateManager;
+        stateManager.selectedObjectEvent += updateSelectedObject;
+    }
+
+    void Update()
+    {
+        transform.LookAt(target);
+    }
+
+    private void OnApplicationQuit()
+    {
+        stateManager.selectedObjectEvent -= updateSelectedObject;
+    }
+
+    void updateSelectedObject(GameObject value)
+    {
+        _selectedObject = value;
+    }
+
+    public void receiveHit(RaycastHit hit)
+    {
+        _selectedObject.transform.parent = init.deletedProps.transform;
```

Comment:
make a note above with 'TODO: flesh out what to do with deleted props'

Reply:
got it